pages/linux/zgrep.md

```diff
@@ -0,0 +1,32 @@
+# zgrep
+
+> Grep pattern(s) of text (strings/regex) from files within compressed file (equivalent to grep -Z).
+> Syntax: zgrep [grep_options] pattern file.
+
+- Grep pattern from compressed file (case-sensitive):
```

Comment:
```suggestion
- Grep a pattern in a compressed file (case-sensitive):
```
Should be similar to this in other examples, too.

Reply:
changed